wagtail/models/__init__.py

```diff
@@ -2698,12 +2685,6 @@ def page_revisions(self):
     def not_page_revisions(self):
         return self.exclude(self.page_revisions_q())
 
-    def submitted(self):
```

Comment:
We also need to remove `PageRevisionsManager.submitted()` and `SubmittedRevisionsManager` a few lines below